package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "openhim-core",
   "description": "The OpenHIM core application that provides logging and routing of http requests",
-  "version": "7.0.0",
+  "version": "7.0.1",
```

Comment:
I think this should be 7.1.0 - this adds new functionality to OpenHIM without affecting the previous versions. It's not really a patch as this was not a defined feature of the OpenHIM AFAIK

Reply:
NVM - I just checked and this is definitely something we were supposed to have supported - so 7.0.1 👍🏼